File: 3. bolum/while/go/yillikfaizhesaplama.go
package main

import "fmt"

func main() {
	var para float64 = 1000
	var ay int = 0

	for para < 5000 {
		para += para * 0.08
		ay++
	}
	fmt.Printf("Paraniz %d ay sonra %f tl olur", ay, para)
}

File: 3. bolum/while/c/yillikfaizhesaplama.c
main()
{
    float para = 1000;
    int ay = 0;
    while (para < 5000)
    {
        para += para * 0.08;
        ay++;

        printf("Ay : %d ,  Para : %f \n ", ay, para);
    }
    printf("Paraniz %d ay sonra %f tl olur.", ay, para);
}